samples/js/flickr/slurp/src/flickr.js

```diff
@@ -0,0 +1,131 @@
+// @flow
```

Comment:
I added a class to keep track of the state (instead of globals). Then there is the whole two stage auth-token to access-token dance, which include getting the verification code.